### PR TITLE
Baazar horizontal items

### PR DESCRIPTION
**Pull request by an-uhryn**

### CODE REVIEW

File: src/components/Items/common/HorizontalLink/styles.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            '.narrowed &': {
+                fontSize: 14,
+
+                '&:hover': {
+                    textDecoration: 'underline',
+                }
+            }
+        }
+    },
+
```
Comment (dudendy) [inline src/components/Items/common/HorizontalLink/styles.js:46]: please remove all empty lines between css properties

File: src/components/Gotchi/Gotchi.js [hunk 2]
```diff
@@ -178,74 +139,33 @@ export default function Gotchi({ gotchi, title, narrowed, renderSvgByStats, rend
...
     function renderSection(value) {
         if (typeof value === 'string') return gotchiSections[value];
 
         return (
             Object.keys(value).map((key) => (
-                gotchiSections[key](value[key].map( item => (
-                    renderSection(item)
-                )))
+                gotchiSections[key](
+                    value[key].map( item => (
```
Comment (dmutrokot) [inline src/components/Gotchi/Gotchi.js:159]: extra space before `item`

File: src/components/Gotchi/GotchiHorizontal.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+            return (
+                <div className={classes.gotchiBadges} key={`${gotchi.id}-badges`}>
+                    {children}
+                </div>
+            );
+        },
+
+        imageCell(children) {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:28]: extra whitespace

File: src/components/Gotchi/GotchiHorizontal.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+            return (
+                <div className={classes.gotchiImageCell} key={`${gotchi.id}-imageCell`}>
+                    {children}
+                </div>
+            );
+        },
+
+        // body
+        traitsCell(children) {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:38]: extra whitespace

File: src/components/Gotchi/GotchiHorizontal.js [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+            return (
+                <div key={`${gotchi.id}-traitsCell`} className={classes.gotchiTraitsCell}>
+                    {children}
+                </div>
+            )
+        },
+
+        priceCell(children) {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:47]: extra whitespace

File: src/components/Gotchi/GotchiHorizontal.js [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+            return (
+                <div key={`${gotchi.id}-priceCell`} className={classes.gotchiPriceCell}>
+                    {children}
+                </div>
+            )
+        },
+
+        get id() {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:56]: extra whitespace

File: src/components/Gotchi/GotchiHorizontal.js [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+                <GotchiId
+                    gotchi={gotchi}
+                    title={title}
+                    key={`${gotchi.id}-id`}
+                />
+            );
+        },
+
+        get collateral() {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:67]: extra whitespace

File: src/components/Gotchi/GotchiHorizontal.js [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+            return (
+                <GotchiCollateral
+                    gotchi={gotchi}
+                    key={`${gotchi.id}-collateral`}
+                />
+            );
+        },
+
+        get level() {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:77]: extra whitespace

File: src/components/Gotchi/GotchiHorizontal.js [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+                    toNextLevel={gotchi.toNextLevel}
+                    experience={gotchi.experience}
+                    size={25}
+                    key={`${gotchi.id}-level`}
+                />
+            )
+        },
+
+        get mainTraits() {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:90]: extra whitespace

File: src/components/Gotchi/GotchiHorizontal.js [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+            return (
+                <GotchiMainTraits
+                    gotchi={gotchi}
+                    key={`${gotchi.id}-mainTraits`}
+                />
+            );
+        },
+
+        get numericTraits() {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:100]: extra whitespace

File: src/components/Gotchi/GotchiHorizontal.js [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+                <GotchiTraitsHighlight
+                    traits={gotchi.numericTraits}
+                    currentTraits={gotchi.modifiedNumericTraits}
+                    key={`${gotchi.id}-numericTraits`}
+                />
+            )
+        },
+
+        get name() {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:111]: extra whitespace

File: src/components/Gotchi/GotchiHorizontal.js [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+            return (
+                <GotchiName
+                    gotchi={gotchi}
+                    key={`${gotchi.id}-name`}
+                />
+            );
+        },
+
+        get svg() {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:121]: extra whitespace

File: src/components/Gotchi/GotchiHorizontal.js [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+                    gotchi={gotchi}
+                    renderSvgByStats={renderSvgByStats}
+                    key={`${gotchi.id}-svg`}
+                />
+            );
+        },
+
+        // price
+        get price() {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:133]: extra whitespace

File: src/components/Items/Wearable/styles.js [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        minHeight: '100px',
+        width: '100%',
+        marginBottom: 4,
+        '.tooltip-wearable &': {
+            minHeight: 60
+        },
+
```
Comment (dudendy) [inline src/components/Items/Wearable/styles.js:14]: please remove all empty lines between css properties

File: src/components/Gotchi/GotchiHorizontal.js [hunk 15]
```diff
@@ -0,0 +1,159 @@
...
+            return (
+                <HorizontalPrice item={item} key={`${gotchi.id}-gotchi-price`} label='Sold for' />
+            )
+        }
+    }
+
+    function renderSection(value) {
+        if (typeof value === 'string') return gotchiSections[value];
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:141]: please wrap `return` statement into `{}`

File: src/components/Gotchi/GotchiHorizontal.js [hunk 16]
```diff
@@ -0,0 +1,159 @@
...
+
+        return (
+            Object.keys(value).map( (key) => (
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:144]: extra space before `key`

File: src/components/Gotchi/GotchiHorizontal.js [hunk 17]
```diff
@@ -0,0 +1,159 @@
...
+                gotchiSections[key](value[key].map( item => (
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:145]: extra space before `item`

File: src/components/Gotchi/GotchiHorizontal.js [hunk 18]
```diff
@@ -0,0 +1,159 @@
...
+                    renderSection(item)
+                )))
+            ))
+        )
+    }
+
+    return (
+        <div className={classNames(classes.gotchi, `haunt${gotchi.hauntId}`, narrowed && 'narrowed', 'horizontal')}>
+            {render.map( (name) => {
```
Comment (dmutrokot) [inline src/components/Gotchi/GotchiHorizontal.js:154]: extra space before `name`

File: src/components/Gotchi/common/GotchiBadges.js [hunk 19]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styles from '../styles';
```
Comment (dmutrokot) [inline src/components/Gotchi/common/GotchiBadges.js:2]: please separate

File: src/components/Gotchi/common/GotchiMainTraits.js [hunk 20]
```diff
@@ -0,0 +1,42 @@
...
+
+    return (
+        <div className={classNames(classes.gotchiMainTraits, classes.gotchiTraits)}>
+            <div className={classes.gotchiTraitsInner}>
+                <HighlightNumber type={calculateRarityType(gotchi.modifiedRarityScore)}>
+                    <p className={classes.mainVal}>
+                        🏆{gotchi.modifiedRarityScore}
+
+                        <span className={classes.defaultVal}>
+                                    ({gotchi.baseRarityScore})
```
Comment (dmutrokot) [inline src/components/Gotchi/common/GotchiMainTraits.js:27]: please format

File: src/components/Gotchi/common/GotchiRewards.js [hunk 21]
```diff
@@ -0,0 +1,55 @@
...
+    return (
+        gotchi.reward || gotchi.reward === 0 ? (
+            <div className={classes.rankBox}>
+                {gotchi.reward > 0 ? (
+                    <Tooltip
+                        title={
+                            <>
+                                {gotchi.rewardStats.map((item, index) => {
+                                    return item.reward !== 0 ? (
+                                        <p key={index}>
```
Comment (dmutrokot) [inline src/components/Gotchi/common/GotchiRewards.js:22]: please avoid using index as a `key` if possible

File: src/components/Gotchi/styles.js [hunk 22]
```diff
@@ -353,14 +387,28 @@ const GotchiWareableLineStyles = makeStyles( theme => ({
...
         flexWrap: 'wrap',
-        minHeight: 26
+        minHeight: 26,
+
+        '.horizontal &': {
+            marginTop: theme.spacing(1)
+        }
+    },
+    gotchiMainTraits: {
+
```
Comment (dmutrokot) [inline src/components/Gotchi/styles.js:404]: extra whitespace

File: src/components/Items/Portal/PortalHorizontal.js [hunk 23]
```diff
@@ -0,0 +1,91 @@
...
+import CardStats from '../common/CardStats/CardStats';
+import HorizontalPrice from '../common/HorizontalPrice/HorizontalPrice';
+import HorizontalLink from '../common/HorizontalLink/HorizontalLink';
+import styles from './../styles';
+
+export default function PortalHorizontal({ portal, render }) {
+    const classes = styles();
+
+    const gotchiSections = {
+
```
Comment (dmutrokot) [inline src/components/Items/Portal/PortalHorizontal.js:15]: extra whitespace

File: src/components/Items/Portal/PortalHorizontal.js [hunk 24]
```diff
@@ -0,0 +1,91 @@
...
+        imageCell(children) {
+
```
Comment (dmutrokot) [inline src/components/Items/Portal/PortalHorizontal.js:17]: extra whitespace

File: src/components/Items/Portal/PortalHorizontal.js [hunk 25]
```diff
@@ -0,0 +1,91 @@
...
+            return (
+                <div className={classes.portalImageCell}>
+                    {children}
+                </div>
+            )
+        },
+
+        infoCell(children) {
+
```
Comment (dmutrokot) [inline src/components/Items/Portal/PortalHorizontal.js:26]: extra whitespace

File: src/components/Items/Portal/PortalHorizontal.js [hunk 26]
```diff
@@ -0,0 +1,91 @@
...
+            return (
+                <div className={classes.portalInfoCell} key={`${portal.id}-portal-infoCell`}>
+                    {children}
+                </div>
+            )
+        },
+
+        priceCell(children) {
+
```
Comment (dmutrokot) [inline src/components/Items/Portal/PortalHorizontal.js:35]: extra whitespace

File: src/components/Items/Portal/PortalHorizontal.js [hunk 27]
```diff
@@ -0,0 +1,91 @@
...
+        },
+
+        // price
+        get price() {
+            return <HorizontalPrice label='Sold for ' item={portal} key={`${portal.id}-portal-price`} />
+        }
+    }
+
+    function renderSection(value) {
+        if (typeof value === 'string') return gotchiSections[value];
```
Comment (dmutrokot) [inline src/components/Items/Portal/PortalHorizontal.js:73]: please wrap `return` into `{}`

File: src/components/Items/Portal/PortalHorizontal.js [hunk 28]
```diff
@@ -0,0 +1,91 @@
...
+
+        return (
+            Object.keys(value).map( (key) => (
```
Comment (dmutrokot) [inline src/components/Items/Portal/PortalHorizontal.js:76]: extra space before `key`

File: src/components/Items/Portal/PortalHorizontal.js [hunk 29]
```diff
@@ -0,0 +1,91 @@
...
+                gotchiSections[key](value[key].map( item => (
```
Comment (dmutrokot) [inline src/components/Items/Portal/PortalHorizontal.js:77]: extra space before `item`

File: src/components/Items/Portal/PortalHorizontal.js [hunk 30]
```diff
@@ -0,0 +1,91 @@
...
+                    renderSection(item)
+                )))
+            ))
+        )
+    }
+
+    return (
+        <div className={classNames(classes.horizontalCard, `haunt${portal.portal.hauntId}`)}>
+            {render.map( (name) => {
```
Comment (dmutrokot) [inline src/components/Items/Portal/PortalHorizontal.js:86]: extra space before `name `

File: src/components/Gotchi/GotchiHorizontal.js [hunk 31]
```diff
@@ -0,0 +1,159 @@
...
+import GotchiTraitsHighlight from './GotchiTraitsHighlight';
+import HorizontalPrice from '../Items/common/HorizontalPrice/HorizontalPrice';
+import styles from './styles';
+
+export default function GotchiHorizontal({ gotchi, item, title, narrowed, renderSvgByStats, render }) {
+    const classes = styles();
+
+    const gotchiSections = {
+        badges: (children) => {
+
```
Comment (dudendy) [inline src/components/Gotchi/GotchiHorizontal.js:19]: please format functions same as in `<Gotchi>` component

```
funcitonName(prop) {
    return (
        content
    );
},
```

File: src/components/Items/Portal/PortalImage.js [hunk 32]
```diff
@@ -0,0 +1,25 @@
...
+    };
+
+    return (
+        <img className={classes.portalImage}
+             style={{'maxWidth': '100px'}}
+             src={
+                portal.portal.hauntId === '1' ?
+                (portal.category === '0' ? sealedPortal : openPortal) :
+                (portal.category === '0' ? h2SealedPortal : h2OpenPortal)
+        } alt='Portal' />
```
Comment (dmutrokot) [inline src/components/Items/Portal/PortalImage.js:23]: please format

File: src/components/Items/Ticket/TicketHorizontal.js [hunk 33]
```diff
@@ -0,0 +1,94 @@
...
+        imageCell(children) {
+            return (
+                <div className={classes.ticketImageCell}>
+                    {children}
+                </div>
+            )
+        },
+
+        infoCell(children) {
+
```
Comment (dmutrokot) [inline src/components/Items/Ticket/TicketHorizontal.js:28]: extra whitespace

File: src/components/Items/Ticket/TicketHorizontal.js [hunk 34]
```diff
@@ -0,0 +1,94 @@
...
+            return (
+                <div className={classes.ticketInfoCell} key={`${ticket.id}-infoCell`}>
+                    {children}
+                </div>
+            )
+        },
+
+        priceCell(children) {
+
```
Comment (dmutrokot) [inline src/components/Items/Ticket/TicketHorizontal.js:37]: extra whitespace

File: src/components/Items/Ticket/TicketHorizontal.js [hunk 35]
```diff
@@ -0,0 +1,94 @@
...
+        get cardStats() {
+            return (
+                <CardStats itemStats={`Quantity: ${ticket.quantity}`} />
+            )
+        },
+
+        get cardName() {
+            return (
+                <CardName itemName={`${ticket.name || itemUtils.getItemRarityName(ticket.erc1155TypeId)} ticket`}
+                                  itemRarity={itemUtils.getItemRarityName(ticket.erc1155TypeId)} item={ticket} />
```
Comment (dmutrokot) [inline src/components/Items/Ticket/TicketHorizontal.js:54]: please put props from newline

File: src/components/Gotchi/common/GotchiCollateral.js [hunk 36]
```diff
@@ -0,0 +1,19 @@
...
+import graphUtils from 'utils/graphUtils';
+
+import styles from '../styles';
+
+export default function GotchiCollateral({ gotchi }) {
+    const classes = styles();
+    const collateral = graphUtils.getCollateralName(gotchi.collateral);
+
+    return (
+        <Tooltip title={collateral} classes={{ tooltip: classes.customTooltip }} enterTouchDelay={0} placement='top' followCursor>
```
Comment (dudendy) [inline src/components/Gotchi/common/GotchiCollateral.js:13]: let's avoid long code like this and split props one by line

File: src/components/Gotchi/common/GotchiId.js [hunk 37]
```diff
@@ -0,0 +1,17 @@
...
+
+import { Tooltip } from '@mui/material';
+
+import styles from '../styles';
+
+export default function GotchiId({ gotchi, title }) {
+    const classes = styles();
+
+    return (
+        <Tooltip title={`Haunt ${gotchi.hauntId}`} classes={{ tooltip: classes.customTooltip }} enterTouchDelay={0} placement='top' followCursor>
```
Comment (dudendy) [inline src/components/Gotchi/common/GotchiId.js:11]: let's avoid long code like this and split props one by line

File: src/components/Items/Ticket/TicketHorizontal.js [hunk 38]
```diff
@@ -0,0 +1,94 @@
...
+        },
+
+        // price
+        get price() {
+            return <HorizontalPrice label='Sold for ' item={ticket} key={`${ticket.id}-ticket-price`} />
+        }
+    }
+
+    function renderSection(value) {
+        if (typeof value === 'string') return gotchiSections[value];
```
Comment (dmutrokot) [inline src/components/Items/Ticket/TicketHorizontal.js:76]: please wrap `return` into `{}`

File: src/components/Items/Ticket/TicketHorizontal.js [hunk 39]
```diff
@@ -0,0 +1,94 @@
...
+
+        return (
+            Object.keys(value).map( (key) => (
```
Comment (dmutrokot) [inline src/components/Items/Ticket/TicketHorizontal.js:79]: extra space before `key`

File: src/components/Items/Ticket/TicketHorizontal.js [hunk 40]
```diff
@@ -0,0 +1,94 @@
...
+                gotchiSections[key](value[key].map( item => (
```
Comment (dmutrokot) [inline src/components/Items/Ticket/TicketHorizontal.js:80]: extra space before `item`

File: src/components/Items/Ticket/TicketHorizontal.js [hunk 41]
```diff
@@ -0,0 +1,94 @@
...
+                    renderSection(item)
+                )))
+            ))
+        )
+    }
+
+    return (
+        <div className={classNames(classes.horizontalCard, ticket.name || itemUtils.getItemRarityName(ticket.erc1155TypeId) )}>
+            {render.map( (name) => {
```
Comment (dmutrokot) [inline src/components/Items/Ticket/TicketHorizontal.js:89]: extra space before `name`

File: src/components/Items/Portal/PortalHorizontal.js (hunk 24 above)
Comment (dudendy) [inline src/components/Items/Portal/PortalHorizontal.js:17]: please format functions same as in `<Gotchi>` component

```
funcitonName(prop) {
    return (
        content
    );
},
```

File: src/components/Items/Wearable/WearableHorizontal.js [hunk 42]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import WearableImage from './WearableImage';
+import HorizontalPrice from '../common/HorizontalPrice/HorizontalPrice';
+import HorizontalLink from '../common/HorizontalLink/HorizontalLink';
+import CardName from '../common/CardName/CardName';
+import CardStats from '../common/CardStats/CardStats';
+import styles from './../styles';
+import itemUtils from 'utils/itemUtils';
```
Comment (dmutrokot) [inline src/components/Items/Wearable/WearableHorizontal.js:10]: it should goes first in this case in imports statements, as it's absolute path and above imports are relative

File: src/components/Items/Wearable/WearableHorizontal.js [hunk 43]
```diff
@@ -0,0 +1,90 @@
...
+        statsCell(children) {
+            return (
+                <div className={classes.wearableStatsCell} key={`${wearable.id}-statsCell`}>
+                    {children}
+                </div>
+            )
+        },
+
+        priceCell(children) {
+
```
Comment (dmutrokot) [inline src/components/Items/Wearable/WearableHorizontal.js:35]: extra whitespace

File: src/components/Items/Ticket/TicketHorizontal.js (hunk 33 above)
Comment (dudendy) [inline src/components/Items/Ticket/TicketHorizontal.js:28]: please format functions same as in `<Gotchi>` component

```
funcitonName(prop) {
    return (
        content
    );
},
```

File: src/components/Items/Wearable/WearableHorizontal.js [hunk 44]
```diff
@@ -0,0 +1,90 @@
...
+        },
+
+        // price
+        get price() {
+            return <HorizontalPrice label='Sold for ' item={wearable} key={`${wearable.id}-price`} />
+        }
+    }
+
+    function renderSection(value) {
+        if (typeof value === 'string') return gotchiSections[value];
```
Comment (dmutrokot) [inline src/components/Items/Wearable/WearableHorizontal.js:71]: please wrap `return` into `{}`

File: src/components/Items/Wearable/WearableHorizontal.js (hunk 43 above)
Comment (dudendy) [inline src/components/Items/Wearable/WearableHorizontal.js:35]: please format functions same as in `<Gotchi>` component

```
funcitonName(prop) {
    return (
        content
    );
},
```

File: src/components/Items/Wearable/WearableHorizontal.js [hunk 45]
```diff
@@ -0,0 +1,90 @@
...
+
+        return (
+            Object.keys(value).map( (key) => (
```
Comment (dmutrokot) [inline src/components/Items/Wearable/WearableHorizontal.js:74]: extra space before `key`

File: src/components/Items/Wearable/WearableHorizontal.js [hunk 46]
```diff
@@ -0,0 +1,90 @@
...
+                gotchiSections[key](value[key].map( item => (
```
Comment (dmutrokot) [inline src/components/Items/Wearable/WearableHorizontal.js:75]: extra space before `item`

File: src/components/Items/Wearable/WearableHorizontal.js [hunk 47]
```diff
@@ -0,0 +1,90 @@
...
+                    renderSection(item)
+                )))
+            ))
+        )
+    }
+
+    return (
+        <div className={classNames(classes.horizontalCard, itemUtils.getItemRarityById(wearable.erc1155TypeId))}>
+            {wearable.rarity}
+            {render.map( (name) => {
```
Comment (dmutrokot) [inline src/components/Items/Wearable/WearableHorizontal.js:85]: extra space before `name`

File: src/components/Gotchi/styles.js [hunk 48]
```diff
@@ -25,12 +23,29 @@ const styles = makeStyles( theme => ({
 
         '&.narrowed': {
             background: 'none'
+        },
+
```
Comment (dudendy) [inline src/components/Gotchi/styles.js:27]: please remove all empty lines between css properties

File: src/components/Items/common/CardName/CardName.js [hunk 49]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Typography } from '@mui/material';
+import classNames from 'classnames';
```
Comment (dmutrokot) [inline src/components/Items/common/CardName/CardName.js:3]: please separate from core imports as it is third party

File: src/components/Items/common/HorizontalPrice/styles.js [hunk 50]
```diff
@@ -0,0 +1,22 @@
...
+    priceRoot: {
+        '& img': {
+            width: 25
+        },
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+        fontSize: 16,
+        fontWeight: 800,
+
```
Comment (dudendy) [inline src/components/Items/common/HorizontalPrice/styles.js:13]: please remove all empty lines between css properties

**Review by dudendy — CHANGES_REQUESTED** [review]

File: src/components/Items/styles.js [hunk 51]
```diff
@@ -47,12 +47,182 @@ const styles = makeStyles( theme => ({
...
+        '&.spacious': {
+            backgroundColor: alpha(theme.palette.realm.spacious, .15)
+        },
+        '&.partner': {
+            backgroundColor: alpha(theme.palette.realm.partner, .15)
+        },
+        '&.haunt1': {
+            backgroundColor: alpha(theme.palette.haunts.h1, .15),
+        },
+
```
Comment (dmutrokot) [inline src/components/Items/styles.js:96]: extra whitespace

File: src/components/Items/styles.js [hunk 52]
```diff
@@ -47,12 +47,182 @@ const styles = makeStyles( theme => ({
...
+        '.mythical &': {
+            backgroundColor: alpha(theme.palette.rarity.mythical, .15)
+        }
+    },
+    wearableStatsCell: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-around',
+        // alignItems: 'center',
```
Comment (dmutrokot) [inline src/components/Items/styles.js:135]: please remove if not needed

File: src/components/Items/styles.js [hunk 53]
```diff
@@ -47,12 +47,182 @@ const styles = makeStyles( theme => ({
...
+
+        '.haunt2 &': {
+            backgroundColor: alpha(theme.palette.haunts.h2, .15),
+        },
+    },
+    portalInfoCell: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-around',
+        // alignItems: 'center',
```
Comment (dmutrokot) [inline src/components/Items/styles.js:164]: please remove if not needed

File: src/components/Items/styles.js [hunk 54]
```diff
@@ -47,12 +47,182 @@ const styles = makeStyles( theme => ({
...
+
+        '.drop &': {
+            backgroundColor: alpha(theme.palette.rarity.drop, .05)
+        }
+    },
+    ticketInfoCell: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-around',
+        // alignItems: 'center',
```
Comment (dmutrokot) [inline src/components/Items/styles.js:213]: please remove if not needed

File: src/pages/Baazaar/components/BaazaarSidebar/styles.js [hunk 55]
```diff
@@ -6,7 +6,9 @@ const styles = makeStyles(theme => ({
         padding: 24,
         borderRadius: 8,
         background: theme.palette.background.paper,
-        maxWidth: 500,
+        width: 300,
+        // position: 'sticky',
```
Comment (dmutrokot) [inline src/pages/Baazaar/components/BaazaarSidebar/styles.js:10]: please remove if not needed

File: src/pages/Baazaar/components/BaazaarSortingBody.js [hunk 56]
```diff
@@ -0,0 +1,174 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
```
Comment (dmutrokot) [inline src/pages/Baazaar/components/BaazaarSortingBody.js:2]: should be separated to third party imports

File: src/pages/Baazaar/components/BaazaarSortingBody.js [hunk 57]
```diff
@@ -0,0 +1,174 @@
...
+    }
+
+    return (
+        <div className={classes.baazaarBody}>
+            <div className={classNames(classes.baazaarListItems, selectedGoodsType === listingTypes.activity ? 'horizontal' : '')}>
+                {
+                    // eslint-disable-next-line array-callback-return
+                    goods.map((item, index) => {
+                        if (selectedGoodsType === listingTypes.activity) {
+                            return <div key={index} className={classes.baazaarListItem}>
```
Comment (dmutrokot) [inline src/pages/Baazaar/components/BaazaarSortingBody.js:62]: please avoid setting `key` as `index` if possible

File: src/pages/Baazaar/components/BaazaarSortingBody.js [hunk 58]
```diff
@@ -0,0 +1,174 @@
...
+                                                    },
+                                                    {
+                                                        priceCell: ['price']
+                                                    }
+                                                ]
+                                            }
+                                        />
+                                    )
+                                }
+                                {/*{*/}
```
Comment (dmutrokot) [inline src/pages/Baazaar/components/BaazaarSortingBody.js:127]: please remove if not needed

File: src/pages/Baazaar/components/BaazaarSortingBody.js [hunk 59]
```diff
@@ -0,0 +1,174 @@
...
+                                {/*    (item.__typename === 'ERC721Listing' && item.category === '4') && <Parcel parcel={{...item.parcel, priceInWei: item.priceInWei, tokenId: item.tokenId, baazaarId: item.id}} isBaazaarCard={true}/>*/}
+                                {/*}*/}
+                            </div>
+                        } else {
+                            return <div key={index} className={classes.baazaarListItem}>
```
Comment (dmutrokot) [inline src/pages/Baazaar/components/BaazaarSortingBody.js:132]: please avoid set `index` to `key` if possible

File: src/pages/Baazaar/components/BaazaarSortingBody.js [hunk 60]
```diff
@@ -0,0 +1,174 @@
...
+                        nextPageVisibility={goods.length < limit}
+                        onNextPageClick={onNextPageClick}
+                        onPrevPageClick={onPrevPageClick}
+                    /> :
+                    <Typography className={classes.noGoods} variant='caption'>
+                        Spooky Market has no such goods :(
+                    </Typography>
+                }
+            </div>
+
```
Comment (dmutrokot) [inline src/pages/Baazaar/components/BaazaarSortingBody.js:167]: extra whitespace

File: src/pages/Baazaar/components/BaazaarSortingBody.js [hunk 61]
```diff
@@ -0,0 +1,174 @@
...
+                                    (item.__typename === 'ERC721Listing' && item.category === '4') && <Parcel parcel={{...item.parcel, priceInWei: item.priceInWei, tokenId: item.tokenId, baazaarId: item.id}} isBaazaarCard={true}/>
+                                }
+                            </div>
+                        }
+                    })
+                }
+            </div>
+            <div className={classes.pagination}>
+                {
+                    goods.length ? <Pagination
```
Comment (dmutrokot) [inline src/pages/Baazaar/components/BaazaarSortingBody.js:155]: please move `Pagination` onto new line

**Review by dmutrokot — CHANGES_REQUESTED** [review]

File: src/pages/Baazaar/components/BaazaarSidebar/styles.js [hunk 62]
```diff
@@ -6,7 +6,9 @@ const styles = makeStyles(theme => ({
...
+        // top: 94,
```
[the remark this answers is not in the document]
Comment (Vladislav94) [reply src/pages/Baazaar/components/BaazaarSidebar/styles.js:11]: fixed

File: src/components/Items/common/HorizontalLink/styles.js (hunk 1 above)
Comment (Vladislav94) [reply src/components/Items/common/HorizontalLink/styles.js:46]: done

File: src/components/Items/Wearable/styles.js (hunk 14 above)
Comment (Vladislav94) [reply src/components/Items/Wearable/styles.js:14]: done

File: src/components/Gotchi/common/GotchiCollateral.js (hunk 36 above)
Comment (Vladislav94) [reply src/components/Gotchi/common/GotchiCollateral.js:13]: done

File: src/components/Items/Portal/PortalHorizontal.js (hunk 24 above)
Comment (Vladislav94) [reply src/components/Items/Portal/PortalHorizontal.js:17]: done

File: src/components/Items/Ticket/TicketHorizontal.js (hunk 33 above)
Comment (Vladislav94) [reply src/components/Items/Ticket/TicketHorizontal.js:28]: done

File: src/components/Items/Wearable/WearableHorizontal.js (hunk 43 above)
Comment (Vladislav94) [reply src/components/Items/Wearable/WearableHorizontal.js:35]: done

File: src/components/Gotchi/styles.js (hunk 48 above)
Comment (Vladislav94) [reply src/components/Gotchi/styles.js:27]: done

File: src/components/Items/common/HorizontalPrice/styles.js (hunk 50 above)
Comment (Vladislav94) [reply src/components/Items/common/HorizontalPrice/styles.js:13]: done

File: src/components/Gotchi/Gotchi.js (hunk 2 above)
Comment (Vladislav94) [reply src/components/Gotchi/Gotchi.js:159]: done

File: src/components/Gotchi/GotchiHorizontal.js (hunk 3 above)
Comment (Vladislav94) [reply src/components/Gotchi/GotchiHorizontal.js:28]: done

File: src/components/Gotchi/GotchiHorizontal.js (hunk 4 above)
Comment (Vladislav94) [reply src/components/Gotchi/GotchiHorizontal.js:38]: done

File: src/components/Gotchi/GotchiHorizontal.js (hunk 5 above)
Comment (Vladislav94) [reply src/components/Gotchi/GotchiHorizontal.js:47]: done

**Review by dudendy — APPROVED** [review]

**Pull request merged**